README.md

```diff
@@ -0,0 +1,106 @@
+# Django Url-Shortener App
+
+## Setting up your Development Environment ##
+1. First, fork this repository to your account.
+
+2. Create a virtual environment on your machine. 
+    ```
+    virtualenv -p python3 env
+    ```
+    We recommend using python3-virtualenv. Any other packages would do fine though.
+
+3. Activate the newly created virtual environment:
+    ```
+    cd env
+    source bin/activate
+    ```
+
+4. Clone this repository (this would make rebasing easier).
+    ```
+    git clone https://github.com/modihere/url-shortener.git
+    ```
+    
+5. Install the dependencies for the project.
+    ```
+    cd website
+    pip3 install -r requirements.txt
+    ```
+    
+6. Run the migrations and collect staticfiles
+    ```
+    python3 manage.py migrate
+    python3 manage.py collectstatic
+    ```
+    
+7. Run the live development server on your machine and test it.
+    ```
+    python3 manage.py runserver
```

Comment:
You can also specify in by giving an "or" that the server can be run at other ports as well just by adding the port number at the end of the command.